public/index.html

```diff
@@ -24,7 +24,7 @@
     <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap" rel="stylesheet">
     <title>ACM @ UCSD</title>
   </head>
-  <body>
+  <body class="light-theme">
```

Comment:
will need to do more stuff here to make dark mode work smoothly. discussed more in themeprovider file, + in overall review as well.